master/reference/calicoctl/setup/etcdv3.md

```diff
@@ -1,136 +1,5 @@
 ---
-title: Configuring calicoctl - etcdv3 datastore
-no_canonical: true
+redirect_to: /master/usage/calicoctl/configure/etcd
```

Comment:
So I understand, we're leaving this just so links continue to direct to the right place?

If so, is there a way we could add a comment in this file to indicate that?

Reply:
Sure, I can add a comment in case the metadata is insufficient....which is could be. It is because we are using page-level directs in this site, so we need pages to do the redirects.